test/functional/admin/legacy_statistics_announcement_tags_controller_test.rb

```diff
@@ -0,0 +1,97 @@
+require "test_helper"
+
+class Admin::LegacyStatisticsAnnouncementTagsControllerTest < ActionController::TestCase
```

Comment:
I think you need to update the non-legacy tests so the user has the correct permission 

```
login_as_preview_design_system_user(:departmental_editor)
```

Reply:
Ah, yes, sorry, good spot. Quite an important change!